Cargo.toml

```diff
@@ -7,6 +7,10 @@ members = [
     "testing/test-runtime",
     "testing/integration-tests",
     "testing/ui-tests",
+    ## This cannot be a workspace dependency, because it requires
```

Comment:
can you move this comment above `exclude = ["testing/wasm-tests"]` instead? 

I found it a little bit hard to understand